avaje-sigma/src/main/java/io/avaje/sigma/core/SigmaContext.java

```diff
@@ -232,4 +238,8 @@ public AWSHttpResponse createResponse() {
     return new AWSHttpResponse(
         status, responseHeaders, multiValueResponseHeaders, body, base64Encoded);
   }
+
+  public boolean handlersSkipped() {
```

Comment:
Looks like this should be part of the HttpContext API.

Reply:
don't care either way, but for what purpose? Any handler that calls  `skipRemainingHandlers` is the one skipping, what use has it for a method that tells you if you have already skipped?

It's not like the skipped handlers have any chance to use it either, it's purely for the internals.